src/pythermalcomfort/models.py

```diff
@@ -2420,6 +2399,27 @@ def use_fans_morris(
     return tipping_point
 
 
+def map_stress_category(t):
```

Comment:
please move this function to `utilities.py`. If possible I want to keep the `models.py` file clean and include in it only the functions that calculate the thermal comfort indices.

Reply:
no problem!